FEM/rf_num_new.cpp

```diff
@@ -698,6 +702,17 @@ ios::pos_type CNumerics::Read(ifstream* num_file)
 			     << "\n";
 			continue;
 		}
+		// automatic damping of Newton scheme
+		if (line_string.find("$AUTOMATIC_NEWTON_DAMPING") != string::npos) // NW
+		{
+			line.str(GetLineFromFile1(num_file));
+			line >> automatic_damping_tolerance; //if NR error decreases by less than this factor, the next step will be dampened
```

Comment:
Convinced. I suggest to first fix the general control of the NR scheme (which tolerances for which convergence measures) though and then an "automatic damping" (two follow-up PRs). All this new keyword does now is transfer the formerly hard-coded damping to input file control.